pkg/config/monitoring.go

```diff
@@ -123,15 +123,15 @@ func (m *Monitoring) GetPrometheusStorageRequest() string {
 	return "50Gi"
 }
 
-func (m *Monitoring) GetTemplateList() []string {
+func (m *Monitoring) GetDashBoards() []string {
 	templateList := []string{
-		"endpointsdetailed.yaml",
-		"endpointsreport.yaml",
-		"endpointssummary.yaml",
-		"resources-by-namespace.yaml",
-		"resources-by-pod.yaml",
-		"cluster-resources.yaml",
-		"critical_slo_alerts.yaml",
+		"endpointsdetailed",
+		"endpointsreport",
+		"endpointssummary",
+		"resources-by-namespace",
+		"resources-by-pod",
+		"cluster-resources",
+		"critical-slo-alerts",
```

Comment:
I notice you refactored the `critical-slo-alerts` syntax here. would it be worth making them all follow the same syntax? e.g. the three endpoints ones? Not sure if that would trigger some upgrade issues, e.g. having to delete ones with the older names?

Reply:
In the previous code, just the file name had underscores. The name inside the yaml definition was "critical-slo-alerts". So i renamed the method to return the dashboard names. Yes, once the name is changed the upgrade doesnt work or in the sense, there might be a stale old dashboard there. Also, the name displayed in the grafana dashboard is not the same as the one we define here. They have pretty names, defined in the "Title" section inside the spec. I would not change the CR names for now.

Reply:
@Rajagopalan-Ranganathan yes I was thinking that could be the case. It's a minor consistency thing so not necessary to fix.